### PR TITLE
Adds Placeholder and PlaceholderColor to Picker

### LINKED ISSUE #3935: Picker/Datepicker/timepicker PlaceholderColor
### Description
There is no property for the placeholdercolor like the other components
### Steps to Reproduce

property to be implemented

### Expected Behavior
mypicker.PlaceholderColor=Colors.Pink;
### Actual Behavior
no implemented

### PR DESCRIPTION
**Pull request by Cheesebaron**
### Description of Change ###

Adds Placeholder and PlaceholderColor to Picker

### Issues Resolved ### 

- fixes #3935

### API Changes ###

 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

Placeholder and PlaceholderColor were not present before

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: Xamarin.Forms.Core/Picker.cs [hunk 1]
```diff
@@ -35,7 +35,11 @@ public class Picker : View, IFontElement, ITextElement, IElementConfiguration<Pi
 		public static readonly BindableProperty FontSizeProperty = FontElement.FontSizeProperty;
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
-		
+
+		public static readonly BindableProperty PlaceholderProperty = PlaceholderElement.PlaceholderProperty;
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Core/Picker.cs]: the `Picker` class should implement `IPlaceholderElement` so it can be selected by CSS

File: Xamarin.Forms.Platform.Android/AppCompat/PickerRenderer.cs [hunk 2]
```diff
@@ -140,7 +148,8 @@ void OnClick()
 				{
 					builder.SetTitle(model.Title ?? "");
 					string[] items = model.Items.ToArray();
-					builder.SetItems(items, (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
+					//builder.SetItems(items, (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
+					builder.SetSingleChoiceItems(items, (int)Element.GetValue(Picker.SelectedIndexProperty), (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.Android/AppCompat/PickerRenderer.cs]: how is this change related to Placeholder properties ?

Comment (Cheesebaron) [reply Xamarin.Forms.Platform.Android/AppCompat/PickerRenderer.cs]: Ah sorry, this shouldn't have been in. I was playing around with some other code.

File: Xamarin.Forms.Core/Picker.cs (hunk 1 above)
Comment (Cheesebaron) [reply Xamarin.Forms.Core/Picker.cs]: Fixed

**Comment by Cheesebaron** [comment]
@kingces95 sorry. Have been a busy with other things. I might get some time the coming weekend to add a UITest.

**Comment by kingces95** [comment]
Please re-open when you get a chance to add the test!

**Comment by Cheesebaron** [comment]
@kingces95 please provide some acceptance criteria for this UITest. What kind of coverage do you want and what exactly do you want covered?
